ten-simple-rules-dockerfiles.Rmd

```diff
@@ -144,6 +162,8 @@ It can provide a starting point for users unfamiliar with writing `Dockerfile`s,
 # 2. Use versioned and automatically built base images {-}
 \rulelabel{2}{rule:base}
 
+[SJE: this feels WAY too long.  can't we say it much shorter, i.e. it is good to pin version numbers if you want a stable archive of reproducible code, or use "latest" to check that your code continues to work as other software environments evolve.  I think having a table here with some recommendations for e.g. python, R and julia, base images would go a long way to practically serve about 80% + of data scienceusers? The bullet pointed list at the end of the section is probably most useful.]
```

Comment:
I don't think we want a table - the state of versions change so quickly it would be outdated before the paper comes out.

Reply:
I agree with @sje30 that it's too long. What seems out of scope for this section is going from tags /versions to talking about automated builds. I would nix that content or move it elsewhere.

Reply:
re: table.  I'm okay about not having it.  But if the table will get out of date, presumably so will the example Dockerfiles that we provide?

Reply:
Most definitely - although they are provided in a repository so (in the case that the repository is kept up to date) the paper linking to them would be as well. The references to tags (e.g., we reference a python tag) will definitely be dated, but I see this as different from a table, because a table is intended to be as a lookup / reference, which we don't want to provide. This is different from mentioning an example in the text that a user won't come back looking for.